tests/e2e/pact/resources/common.ts

```diff
@@ -5,7 +5,8 @@ export const basePath = "/FHIR/R4"
 
 export type ApiMode = "live" | "sandbox"
 export type ApiEndpoint = "prepare" | "process" | "task" | "validate" | "verify-signature" | "metadata" | "tracker"
-export type ApiOperation = "send" | "cancel" | "dispense" | "release" | "return" | "withdraw" | "claim"
+export type ApiOperation = "send" | "cancel" | "dispense" | "dispenseamend" |
```

Comment:
does this need to be a single string `dispenseamend` could we do `dispense-amend` instead?

Reply:
Couldn't do this for the string matching reasons discussed